src/app/file-explorer.js

```diff
@@ -253,8 +254,6 @@ function fileExplorer (appAPI, files) {
     el.className = css.fileexplorer
     element.parentElement.replaceChild(el, element)
     element = el
-    fileFocus(filepath)
-    appAPI.switchToFile(filepath)
```

Comment:
Will this  `switchToFile` done by `fileFocus` or app catches the `fileAdded` event and does the switch itself?